CMakeLists.txt

```diff
@@ -141,6 +141,9 @@ if(MSVC)
     # https://gitlab.kitware.com/cmake/cmake/-/issues/18837
     add_compile_options(/Zc:__cplusplus)
 
+    # https://devblogs.microsoft.com/cppblog/announcing-full-support-for-a-c-c-conformant-preprocessor-in-msvc/
```

Comment:
@okwasniewski why adding this option?

Reply:
Without this the build was failing with a message to add it, here is a discussion about this flag: https://github.com/BabylonJS/bgfx/pull/39#issuecomment-2273805547

Reply:
I guess arcana.cpp is not conformat with the wstring macro. let me see if I can find a fix...

Reply:
build is fine with conformant macro : https://github.com/BabylonJS/BabylonNative/pull/1409
I've done the PR for arcana.cpp https://github.com/microsoft/arcana.cpp/pull/45 @ryantrem 
When the PR is merged, you can update the commit id in this PR @okwasniewski

Reply:
arcana.cpp change is merged

Reply:
@CedricGuillemet thanks for letting me know. Looks like the build is now green!

I think we can merge the PR: 

- https://github.com/BabylonJS/bgfx/pull/40
- Then: https://github.com/BabylonJS/bgfx.cmake/pull/76

And I will update this one with correct git repos.

Reply:
@okwasniewski the bgfx one is merged, please go ahead and update the bgfx.cmake one with commit 207e246af754bfdd0e46229997e4c18dbafa6c04.